net-p2p/bitcoin-core/bitcoin-core-25.2_rc1.ebuild

```diff
@@ -0,0 +1,415 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit autotools bash-completion-r1 check-reqs db-use desktop edo multiprocessing python-any-r1 systemd toolchain-funcs xdg-utils
+
+DESCRIPTION="Reference implementation of the Bitcoin cryptocurrency"
+HOMEPAGE="https://bitcoincore.org/"
+SRC_URI="https://github.com/bitcoin/bitcoin/archive/v${PV/_rc/rc}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN/-core}-${PV/_rc/rc}"
+
+LICENSE="MIT"
+SLOT="0"
+if [[ "${PV}" == *_rc* ]] ; then
+	KEYWORDS=""
```

Comment:
Just don't set it at all here. See https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0105 and pkgcheck also emits `MultipleKeywordsLines`. I'll fix it on merge.

Reply:
Sheesh! I could swear I've seen other ebuilds doing this very thing for release candidates in the past. I expected, if I were simply to comment out the `KEYWORDS` line for the RC, that you'd yell at me for writing something that will have to be changed when the ebuilds are eventually bumped to release versions. The policy rules for ebuilds are so much stricter than the syntax allows! 😖

Reply:
I don't think Sam would have yelled, but as you pointed out, it would have been not ideal. And style rules are always stricter than syntax rules, that is their very nature.

I know that this can be frustrating for contributors to ::gentoo, but a certain level of consistency helps us maintaining the vast amount of ebuilds and keeping the quality up.